development/openstack-dpdk/bundle.yaml

```diff
@@ -89,7 +89,6 @@ services:
       fsid: 5a791d94-980b-11e4-b6f6-3c970e8b1cf7
       monitor-secret: AQAi5a9UeJXUExAA+By9u+GPhl8/XiUQ4nwI3A==
       osd-devices: /dev/sdb
-      osd-reformat: True
```

Comment:
This one should really be updated to use charm-ceph or removed entirely; does it even work!?